### PR TITLE
Appv1

### PR DESCRIPTION
**Pull request by malekch-15**
Backend:

Implemented endpoints for GET, GET by ID, and DELETE.
Added unit tests and integration tests for these endpoints.
Integrated a GlobalExceptionHandler to handle exceptions gracefully.
Frontend:

Developed the RecipeCard component.
Implemented data fetching logic along with the DELETE button functionality.

### CODE REVIEW

File: backend/src/main/java/cookme/controller/RecipesController.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    @GetMapping("/{id}")
+    public Recipes getRecipeWithId(@PathVariable String id) {
+        return recipesService.findRecipesById(id);
+    }
+
+
+    @DeleteMapping("/{id}")
+    public void deleteRecipeWithId(@PathVariable String id) {
+    recipesService.findRecipesById(id);
```
Comment (Kinuy) [inline backend/src/main/java/cookme/controller/RecipesController.java:29]: why do you want to find the recipe here? -> call not necessary

File: backend/src/main/java/cookme/recipesmodel/RecipesModelDto.java
Comment (Kinuy) [inline backend/src/main/java/cookme/recipesmodel/RecipesModelDto.java:1]: Why not naming the model file just RecipesDto?

File: backend/src/main/java/cookme/recipesmodel/Recipes.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package cookme.recipesmodel;
+
+import lombok.With;
+
+import java.util.List;
+
+public record Recipes(@With String id,
+                      String name,
+                      String description,
+                      Integer time,
```
Comment (Kinuy) [inline backend/src/main/java/cookme/recipesmodel/Recipes.java:10]: If you have just minutes its ok, but you are more flexible with double

File: frontend/src/Details/Details.tsx
Comment (Kinuy) [inline frontend/src/Details/Details.tsx:1]: Vertical slices!

**Review by Kinuy — APPROVED** [review]
some naming issues....

File: backend/src/main/java/cookme/recipesmodel/Recipes.java (hunk 2 above)
Comment (malekch-15) [reply backend/src/main/java/cookme/recipesmodel/Recipes.java:10]: thank you i will change it

**Pull request merged**